snmp/datadog_checks/snmp/data/profiles/fortinet-fortigate.yaml

```
@@ -0,0 +1,216 @@
+# Profile for Fortinet FortiGate devices
+extends:
+  - _base.yaml
+  - _generic-router-if.yaml
+
+sysobjectid: 1.3.6.1.4.1.12356.101.1.*
+
+metrics:
+
+  ### CPU
+  - MIB: FORTINET-FORTIGATE-MIB
+    symbol:
+      # Current CPU usage (percentage).
+      - OID: 1.3.6.1.4.1.12356.101.4.1.3
+        name: fgSysCpuUsage
+
+  ### Processors
+  # Only valid for processors types that support these statistics.
+  - MIB: FORTINET-FORTIGATE-MIB
+    table:
+      OID: 1.3.6.1.4.1.12356.101.4.4.2
+      name: fgProcessorTable	
```

```suggestion
      name: fgProcessorTable
```
Looks like the CI is failing because a few lines have tabs at the end